shared/src/main/kotlin/pl/edu/pw/ia/shared/domain/command/PostCommands.kt

```diff
@@ -0,0 +1,36 @@
+package pl.edu.pw.ia.shared.domain.command
+
+import java.util.UUID
+import org.axonframework.modelling.command.TargetAggregateIdentifier
+import pl.edu.pw.ia.shared.domain.model.VoteType
+
+data class CreatePostEvent(
```

Comment:
From the more important things in this PR
I have defined all the commands we have defined in project overview https://www.notion.so/mkuranowski/Project-overview-4dbe2e83b28941f9854ce6f6e1babb65

> - Users can create new threads, where they become a **moderator**.
> - Users create **posts** under multiple **threads.**
> - Users can **vote** a post up or down.
> - Users can edit and delete posts they’ve created.
> - Moderators can remove posts from threads they moderate.
> - Moderators can ban users from posting under a thread they moderate.
> - Moderators can appoint and remove new moderators for threads they moderate.
> - Administrators have the same power as moderators on all threads.
> - Administrators can ban users from posting under every thread.